ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -969,6 +986,34 @@ TFLiteCore::setInputTensorDim (const GstTensorsInfo *info)
   return err;
 }
 
+/**
+ * @brief Replace the interpreter, called by reloadModel
+ *        Check input/output tensors have the same info
+ * @param new_interpreter new interpreter to replace with
+ * @return int 0 if ok, non-zero if error
+ */
+int
+TFLiteCore::replaceInterpreter (TFLiteInterpreter * new_interpreter)
+{
+  int err;
+  TFLiteInterpreter * interpreter_temp = interpreter;
+  if (!gst_tensors_info_is_equal (interpreter->getInputTensorsInfo (),
+          new_interpreter->getInputTensorsInfo ())
+      || !gst_tensors_info_is_equal (interpreter->getOutputTensorsInfo (),
+             new_interpreter->getOutputTensorsInfo ())) {
+    ml_loge ("The model has unmatched tensors info\n");
+    err = -EINVAL;
+    return err;
```

Comment:
You may just use `return -EINVAL` and remove `int err;`

Reply:
Fixed. Thanks